pkg/pipeline/write/write_stdout.go

```diff
@@ -28,7 +30,7 @@ type writeStdout struct {
 func (t *writeStdout) Write(in []interface{}) {
 	log.Debugf("entering writeStdout Write")
 	for _, v := range in {
-		log.Infof("%v", v)
+		fmt.Printf("%s: %v\n", time.Now().Format(time.StampMilli), v)
```

Comment:
When we work with stdout it is not connected to log level we just want to print the output to stdout